content/docs/user-guide/components/reference/atom/mesh.md

```diff
@@ -12,8 +12,88 @@ The **Mesh** component specifies a model to render in the scene. Model assets ar
 [Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
 
 
-## Properties
+## Mesh Component properties
 
-{{< todo issue=https://github.com/o3de/o3de.org/issues/717 >}}
-Add properties.
-{{< /todo >}}
+![mesh-component-base-properties](/images/user-guide/components/reference/atom/mesh/mesh-base-properties-ui.png)
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Model Asset** | Sets the model asset for this component. | Model Asset | None |
+| **Sort Key** | Transparent models are first drawn by sort key, then depth. Use this to force certain transparent models to draw before or after others. | -2,147,483,648 to 2,147,483,647 | `0` |
+| **Exclude from reflection cubemaps** | If enabled, the model will not be visible in baked reflection probe cubemaps. | Boolean | `Disabled` |
+| **Use Forward Pass IBL Specular** | Renders image-based lighting (IBL) specular reflections in the forward pass, by using only the most influential probe (based on the position of the entity) and the global IBL cubemap. It can reduce rendering costs, but is only recommended for static objects that are affected by at most one reflection probe. | Boolean | `Disabled` |
+| **Use ray tracing** | Includes this model in ray tracing calculations. | Boolean | `Enabled` |
+| **LOD Type** | Refer to [LOD Type](#lod-type) below. | `Default`, `Screen Coverage`, `Specific LOD` | `Default` |Default (Highest) |
```

Comment:
> |Default (Highest) |
- Did you mean for this to appear in the **Default** column? Currently, it doesn't appear at all since the table is only four columns. 
- If you do want to keep "(Highest)" in, can you explain what that indicates? Maybe in the description. 
```suggestion
| **LOD Type** | Refer to [LOD Type](#lod-type) below. | `Default`, `Screen Coverage`, `Specific LOD` | `Default` |
```